api-report/merge-tree.api.md

```diff
@@ -406,6 +406,8 @@ export interface IMarkerModifiedAction {
 export interface IMergeBlock extends IMergeNodeCommon {
     // (undocumented)
     assignChild(child: IMergeNode, index: number, updateOrdinal?: boolean): void;
+    // (undocumented)
+    cachedLength?: number;
```

Comment:
Can you document the new property? I know we have a lot of undocumented stuff here, but if we can avoid adding new undocumented stuff it's a step in the right direction. I think the description from the pr details might be enough.

Reply:
This field and the one below do have docs, it's just that I haven't rebuilt the API report. Done in https://github.com/microsoft/FluidFramework/pull/15868/commits/a76a52b6c1d34fbc5f8e948c658eaf0b1fabb2be. The docs for these fields right now just copy the existing docs, though I'm considering updating them for ISegment.